adapters/syslog/syslog.go

```diff
@@ -112,7 +115,7 @@ func (a *SyslogAdapter) Stream(logstream chan *router.Message) {
 	}
 }
 
-func (a *SyslogAdapter) retry(buf []byte, err error) error {
+func (a *Adapter) retry(buf []byte, err error) error {
```

Comment:
It's not required or enforceable. However, if a function is doing something complex or "needs" additional explanation, then I tend to add them in that case.